weed/s3api/AmazonS3.xsd

```diff
@@ -525,6 +525,7 @@
       <xsd:element name="IsTruncated" type="xsd:boolean"/>
       <xsd:element name="Contents" type="tns:ListEntry" minOccurs="0" maxOccurs="unbounded"/>
       <xsd:element name="CommonPrefixes" type="tns:PrefixEntry" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="EncodingType" type="xsd:string"/>
```

Comment:
This is not included in the original xsd. Needs to adjust the steps in the readme file.

Reply:
done